applications/ColossalChat/examples/README.md

```diff
@@ -717,14 +718,29 @@ For DPO training, you only need the preference dataset. Please follow the instru
 
 
 #### Step 2: Training
-You can run the [train_dpo.sh](./examples/training_scripts/train_dpo.sh) to start DPO training. Please refer to the [training configuration](#training-configuration) section for details regarding supported training options.
+You can run the [train_dpo.sh](./examples/training_scripts/train_dpo.sh) to start DPO training. Please refer to the [training configuration](#training-configuration) section for details regarding supported training options. Following the trend of recent research on DPO-like alignment methods, we added option for the user to choose from, including whether to do length normalization , reward shaping and whether to use a reference model in calculating implicit reward. Here are those options,
 
+```
+--beta 0.1 \     # the temperature in DPO loss, Default to 0.1
+--gamma 0.0 \     # the reward target margin in the SimPO paper, Default to 0.
+--disable_reference_model \   # whether to disable the reference model, if set, the implicit reward will be calculated solely from the actor. Default to enable reference model in DPO
+--length_normalization \  # whether to apply length normalization, Default to not use
+```
```

Comment:
I suggest refactoring the SimPO part to a separate loss to make it more clearer. I understand both are preference optimization method so you can share the same DPOTrainer but make it more clearer for different method. For example, add one option for POloss ("dpo" or "simpo"). 

User's may not have clear idea of using parameters here, for example, `disable_reference_model` can trigger SimPO at the backend. Thus, we should clearly explain if you want to use DPO, we can adjust which parameters, if you want to use SimPO, you can use which parameters.

Reply:
will do